app/src/main/java/com/gophillygo/app/data/models/Attraction.java

```diff
@@ -66,10 +71,14 @@ public Attraction(int id, int placeID, String name, boolean accessible, String i
         this.description = description;
         this.priority = priority;
         this.websiteUrl = websiteUrl;
-        this.wideImage = wideImage;
         this.isEvent = isEvent;
-
         this.activities = activities;
+
+        this.wideImage = addHostToPath(wideImage);
```

Comment:
This host path modification is only needed in development. Should the host change, the image references stored in the local database will become invalid if the full path is stored like this, instead being generated dynamically as it was before.

How about reverting to adding the host path in the getter as it was before, and putting a `Build.DEBUG` guard on it so it won't be used in production?

Reply:
See azavea/cac-tripplanner#1046